### ISSUE TITLE
EXP duplication glitch

### ISSUE THREAD

**Issue by GrummDado** [Shorepion/Magica-Infinitum-III#5]
this is caused by a corail tombstone config which let the player keep exp upon death, while the enigmatic amulet, if equipped, will summon an extradimensional vessel which keep exp, it should keep the inventory too but there's the tombstone

https://user-images.githubusercontent.com/20334454/104090769-3fcf4200-5279-11eb-8bba-180332fc0b7e.mp4

how to reproduce:
1. equip the enigmatic amulet which can be obtained when creating a new world
2. get some exp
3. /kill
4. the player won't lose the exp upon death, the tombstone has the inventory and the extradimensional vessel has some exp so the amount of exp increase

possible solution is edit in **tombstone-server.toml**
#Experience loss on death (%) [0..100|default:0]
	#Range: 0 ~ 100
	xp_loss = **100**

**Comment by Shorepion** [comment]
Thanks for reporting!
>possible solution is edit in tombstone-server.toml
#Experience loss on death (%) [0..100|default:0]
#Range: 0 ~ 100
xp_loss = 100>

Problem here is, if the player looses the amulet somehow/havent it equipped when dying, 100% of the levels are lost.
I came up with disabling the Extradimensional Vessel function of the amulet completely and set the xp loss (Corail Tombstone) to 20%.
I think thats a good balance (since in vanilla minecraft you loose ~50% of xp when dying).

**Comment by GrummDado** [comment]
that's fair, thanks you

**Comment by Shorepion** [comment]
> that's fair, thanks you

since I cant send pm within github, Im writing it here:
Are you interested in playing on my dev/testserver for Magica Infinitum?
Its a survival Server with the latest devversion of my modpack, for finding bugs, glitches, exploits, unbalanced things.
You can play how you like and are not to forced to do this or that.
If you are interested, pls join https://discord.com/invite/rN6gzMP5qu for more information :)

**Comment by Shorepion** [comment]
since you cant send pm in github, Im writing it here:

Are you interested in playing on my MI3 survival dev server and helping the modpack to improve by the way?
You can play how like you want and when you like. Its very helpful for me to find bugs, glitches, exploits, unbalanced items,configs...
If you are interested you can contact me on Discord (Shorepion#3489) ✌️
